### ISSUE TITLE
OpenSSL needs urgent update (CVE-2021-3711 and CVE-2021-3712)

### ISSUE THREAD

**Issue by mgutt** [NginxProxyManager/docker-nginx-full#1]
This Debian release contains OpenSSL 1.1.1d, but we need 1.1.1l:
https://www.openssl.org/news/secadv/20210824.txt

![image](https://user-images.githubusercontent.com/10757176/131495302-7b9d6f93-4740-4798-92f1-661156e4d23f.png)

![image](https://user-images.githubusercontent.com/10757176/131499393-d573d0d4-9e43-4eeb-8c10-13677dcdddac.png)

EDIT: Ok, I found out that Debian fixed 1.1.1d, so we don't need to update to 1.1.1l:
https://security-tracker.debian.org/tracker/CVE-2021-3711

Verified as follows:
```
apt list -a openssl
Listing... Done
openssl/oldstable,now 1.1.1d-0+deb10u7 amd64 [installed]
openssl/oldstable 1.1.1d-0+deb10u6 amd64
```

**Comment by jc21** [comment]
Feel free to create a PR to install your required version [here](https://github.com/NginxProxyManager/docker-nginx-full/blob/master/docker/Dockerfile#L94).

**Comment by mgutt** [comment]
>Feel free to create a PR
It's not possible to install 1.1.1l in Debian Buster. But the issue is solved as it already contains the fixed version "1.1.1d-0+deb10u7".
